<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(103, 61, 255, 255), stop:0.505682 rgba(0, 147, 246, 255), stop:1 rgba(49, 58, 255, 255));
font-family: Ruda Regular;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;
color: white;
font-size: 25px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;
color: white;
font-size: 25px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;
color: white;
font-size: 25px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-weight: bold;
font-size: 18pt;
background-color: none;
border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>301</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Изменить напоминание</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="newTextReminder">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255,255,255,40);
font-size: 20px;
font-weight: normal;</string>
       </property>
       <property name="placeholderText">
        <string>Новое название напоминания</string>
       </property>
      </widget>
      <widget class="QPushButton" name="changeButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>301</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ruda Regular</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: white;
font-weight: normal;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;
width: 100px;
height: 50px;
}

QPushButton:hover{
background-color: rgba(255,255,255,40)
}

QPushButton:pressed{
background-color: rgba(255,255,255,70)
}</string>
       </property>
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
